src/components/application_manager/test/include/application_manager/mock_message_helper.h

```diff
@@ -221,10 +221,10 @@ class MockMessageHelper {
                     const std::string& packageName,
                     ApplicationManager& app_man));
   MOCK_METHOD1(SendUnsubscribedWayPoints, bool(ApplicationManager& app_mngr));
-  MOCK_METHOD3(VerifyImage,
+  /*MOCK_METHOD3(VerifyImage,
```

Comment:
@wolfylambova22, why this code is commented? If it's redundant, please, remove it.